imports/plugins/custom/products-review/client/components/Reviews.js

```diff
@@ -0,0 +1,264 @@
+import React, { Component } from "react";
+import ReactStars from "react-stars";
+import { Meteor } from "meteor/meteor";
+import "../stylesheet/style.css";
+import PropTypes from "prop-types";
+import moment from "moment";
+import Pagination from "react-js-pagination";
+// import toastrOption from "../../utils/toastrOption";
+/**
+ * @class Review
+ *
+ * @classdesc user post review
+ *
+ */
+class Reviews extends Component {
+  /**
+   * constructor - contains the constructor
+   *
+   * @param  {object} props the properties of the class component
+   *
+   * @return {void} no return or void
+   *
+   */
+  constructor(props) {
+    super(props);
+    this.state = {
+      review: "",
+      rating: 0,
+      username: "",
+      disableBtn: true,
+      reviews: "",
+      totalCount: 0,
+      totalRating: 0,
+      activePage: 1,
+      paginatedReviews: []
+    };
+    this.onChange = this.onChange.bind(this);
+    this.ratingChanged = this.ratingChanged.bind(this);
+    this.addReview = this.addReview.bind(this);
+    this.reviewList = this.reviewList.bind(this);
+    this.handlePageChange = this.handlePageChange.bind(this);
+  }
+  /**
+ *
+ * @returns { object } updated state
+ * @memberof Ratings
+*/
+  componentDidMount() {
+    const destination = this.props.product._id;
+    let totalRating = 0;
+    Meteor.call("getAllReviews", destination, (error, response) => {
+      if (error) {
+        return error;
+      }
+      response.forEach(eachRating => { totalRating += Number(eachRating.rating);});
+      this.setState({ reviews: response, paginatedReviews: response.slice(0, 5), totalCount: response.length, totalRating });
+    });
+  }
+  /**
+   * @description - handles the onchange event
+   *
+   * @param  {object} event the event for the content field
+   *
+   * @return {void}
+   */
+  onChange(event) {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+    if (event.target.name === "review" && event.target.value.trim() === "") {
+      this.setState({
+        disableBtn: true
+      });
+      return false;
+    }
+    this.setState({
+      disableBtn: false
+    });
+    return true;
+  }
+
+  handlePageChange(pageNumber) {
+    let limit = 5;
+    const offset = (pageNumber - 1) * limit;
+    limit += offset;
+    const paginatedReviews = this.state.reviews.slice(offset, limit);
+    this.setState({ activePage: pageNumber, paginatedReviews });
+  }
+
+  reviewList() {
+    const allReviews = this.state.paginatedReviews;
+    if (allReviews.length === 0) {
+      return (<div className="reviews-contents">No reviews yet.</div>);
+    }
+    return (
+      allReviews.map((review) => (
+        <div key={review._id}>
+          <div className="reviews-contents">
+            <a href="#" className="comment-author" title="Comment Author">
```

Comment:
Use a button or a div instead of the anchor tag so as to fix the hound issue.

Reply:
Done.